back/lib/gem_extensions/que/migrations.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module GemExtensions
+  module Que
+    module Migrations
+      # This extension adapts Que's migrations to make them compatible with Apartment.
+      # The Que database version (an internal version number used by Que to version
+      # changes in how it persists jobs) is stored as a comment on the `que_jobs` table.
+      # However, the original implementation of `db_version` does not expect the database
+      # to contain multiple `que_jobs` tables and, as a result, is returning the version
+      # of one of them indeterministically.
+      module ClassMethods
+        def db_version
+          current_schema_name = Apartment::Tenant.current
```

Comment:
So just for my understanding, the reason why we're using `current` here instead of hard coding this to `public`, is that we want to keep all que tables across all schemas migrated, so when the migrations, that will be executed for every tenant, call this, they get back the right version. Right?

Reply:
Exactly! I considered removing Que from all the schemas except `public`, but this was coming with other problems. When creating new tenants, Apartment would have copied the public schema, including Que objects.